plutus-core/plutus-core/src/PlutusCore/Parser/Builtin.hs

```diff
@@ -74,7 +74,7 @@ conList uniA = inBrackets $ constantOf uniA `sepBy` symbol ","
 
 -- | Parser for pairs.
 conPair :: DefaultUni (Esc a) -> DefaultUni (Esc b) -> Parser (a, b)
-conPair uniA uniB = inParens $ do
+conPair uniA uniB = inParensSpc $ do
```

Comment:
Why is this different?

Reply:
`inParensSpc` is the new `inParens`. Previously, `inParens` consumes whitespaces, while `inParens'` doesn't.